cmd/cmd.go

```diff
@@ -913,6 +959,57 @@ func generateInteractive(cmd *cobra.Command, opts generateOptions) error {
 	}
 }
 
+func normalizeFilePath(fp string) string {
+	// Define a map of escaped characters and their replacements
+	replacements := map[string]string{
+		"\\ ":  " ",  // Escaped space
+		"\\(":  "(",  // Escaped left parenthesis
+		"\\)":  ")",  // Escaped right parenthesis
+		"\\[":  "[",  // Escaped left square bracket
+		"\\]":  "]",  // Escaped right square bracket
+		"\\{":  "{",  // Escaped left curly brace
+		"\\}":  "}",  // Escaped right curly brace
+		"\\$":  "$",  // Escaped dollar sign
+		"\\&":  "&",  // Escaped ampersand
+		"\\;":  ";",  // Escaped semicolon
+		"\\'":  "'",  // Escaped single quote
+		"\\\\": "\\", // Escaped backslash
+		"\\*":  "*",  // Escaped asterisk
+		"\\?":  "?",  // Escaped question mark
+	}
+
+	for escaped, actual := range replacements {
+		fp = strings.ReplaceAll(fp, escaped, actual)
+	}
+	return fp
+}
+
+func extractFileNames(input string) (string, []ImageData, error) {
+	// Regex to match file paths starting with / or ./ and include escaped spaces (\ or %20)
+	// and followed by more characters and a file extension
+	regexPattern := `(?:\./|/)[\S\\ ]+?\.(?:jpg|jpeg|png|svg)\b`
```

Comment:
I have a question for file extension. If the Model is able to describe a jpg, is it difficult to accept a path to a mp3 for a model that could convert the sound to text?

Reply:
I think it could be nice to give the path to a file in the prompt and Ollama could read the data of that path and send it to the Model to be Handle. In the video you provided 2 weeks ego, you show that. I was wondering, why just pics and not mp3 or json.

Reply:
@igorschlum both of those could be possible in the future, but different types of files rely on having support in the library we use to run the LLM.

It also relies on the models themselves to be designed and tuned for those use-cases too. At the moment image-to-text models are easier for us to support.